Week2/homework/maartjes-work.js

```diff
@@ -45,21 +45,28 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
-
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
+const convertDurationsToHour = duties =>
+  duties.map(item => ({
+    name: item.name,
+    duration: item.duration / 60,
+  }));
+const filterTasks = (tasks = []) => tasks.filter(item => item.duration >= 2);
 
-// add code to convert `earnings` to a string rounded to two decimals (euro cents)
+console.log(convertDurationsToHour(maartjesTasks));
+console.log(filterTasks(convertDurationsToHour(maartjesTasks)));
```

Comment:
You could also put line 48-56 within the `computeEarnings()` function, that way it is all nicely self contained in one function.